contrib/db_pools/lib/Cargo.toml

```diff
@@ -52,9 +52,7 @@ features = ["rt_tokio_1"]
 optional = true
 
 [dependencies.mongodb]
-version = "1"
-default-features = false
-features = ["tokio-runtime"]
+version = "2.0.0"
```

Comment:
The only default feature for `mongodb` is `tokio-runtime`.

Reply:
I believe this was actually intentional (for all database dependencies)! This way, `rocket_db_pools` avoids accidentally enabling any other default features that are added in future versions, which not all projects may need/want.